flutter/android/src/main/kotlin/io/sentry/flutter/SentryFlutterPlugin.kt

```diff
@@ -11,19 +14,28 @@ import io.sentry.HubAdapter
 import io.sentry.Sentry
 import io.sentry.SentryEvent
 import io.sentry.SentryLevel
+import io.sentry.android.core.ActivityFramesTracker
 import io.sentry.android.core.AppStartState
+import io.sentry.android.core.LoadClass
 import io.sentry.android.core.SentryAndroid
 import io.sentry.android.core.SentryAndroidOptions
 import io.sentry.protocol.DebugImage
 import io.sentry.protocol.SdkVersion
+import io.sentry.protocol.SentryId
 import java.io.File
+import java.lang.ref.WeakReference
 import java.util.Locale
 import java.util.UUID
 
-class SentryFlutterPlugin : FlutterPlugin, MethodCallHandler {
+@file:Suppress("TooManyFunctions")
```

Comment:
@denrase '@file:' annotations can only be applied before package declaration

Reply:
Interesting, this did work for locally executing `detekt` with gradle.